jujugui/static/gui/src/app/app.js

```diff
@@ -1411,19 +1412,26 @@ YUI.add('juju-gui', function(Y) {
     */
     _setupCharmstore: function(Charmstore) {
       if (this.get('charmstore') === undefined) {
-        var jujuConfig = window.juju_config, charmstoreURL, existingMacaroons;
+
+        var jujuConfig = window.juju_config, charmstoreURL, existingMacaroons,
+            existingCookie;
```

Comment:
Tiny thing: can we put the undefined vars on separate lines? It's a bit clearer that it's not part of an expression being assigned to jujuConfig.

e.g.

```
var jujuConfig = window.juju_config,
    charmstoreURL, existingMacaroons, existingCookie;
```